wifiphisher/pywifiphisher.py

```diff
@@ -595,6 +595,10 @@ def start(self):
                        "rogue Access Point").format(G, W, ap_iface.get_name())
 
             kill_interfering_procs()
+            #randomize the adapters MAC addresses
+            self.network_manager.randomize_interface_mac(ap_iface)
+            self.network_manager.randomize_interface_mac(mon_iface)
```

Comment:
Can we also print that to the user? Something like "[!] Changing MAC address of wlan0 interface to 00:00:00:12:34:56".